extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -88,16 +88,68 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 	let connectionStringSettingName;
 
 	// show the settings from project's local.settings.json if there's an AF functions project
-	// TODO: allow new setting name to get added here and added to local.settings.json
 	if (project) {
-		const settings = await azureFunctionsUtils.getLocalSettingsJson(path.join(path.dirname(project!), constants.azureFunctionLocalSettingsFileName));
-		const existingSettings: string[] = settings.Values ? Object.keys(settings.Values) : [];
+		let settings;
+		try {
+			settings = await azureFunctionsUtils.getLocalSettingsJson(path.join(path.dirname(project!), constants.azureFunctionLocalSettingsFileName));
+		} catch (e) {
+			void vscode.window.showErrorMessage(e);
+			return;
+		}
+
+		let existingSettings = settings?.Values ? Object.keys(settings.Values).map(setting => {
+			return {
+				label: setting
+			} as vscode.QuickPickItem & { isCreateNew?: boolean };
+		}) : [];
```

Comment:
A bit chunky for a ternary.  Maybe initialize as `[]`, then replace with the rest if settings?.Values